tests/test_model.py

```diff
@@ -50,3 +52,12 @@ def test_model():
     def helper():
         Model(mlp4.apply(mlp3.apply(x)))
     assert_raises(ValueError, helper)
+
+
+def test_model_handles_brickless_parameteres():
```

Comment:
Spelling error, but I'll add this as a softball ticket for someone's first CCW PR.